### DISCUSSION TITLE
field "count(*)" not exist

### DISCUSSION THREAD

**Discussion by duncan-dao-90** [milvus-io/milvus#27642] [Q&A and General discussion] [answered]
I'm following document below, to get count number of entities

https://milvus.io/docs/query.md#Count-entities

```
  const result = await milvusClient.query({
    collection_name: `my collection`,
    output_fields: ['count(*)'],
    expr: `my expr`,
  });

  console.log(result);
```

However when I try with my Zilliz cluster, I got below error response

```
{
  status: {
    error_code: 'UnexpectedError',
    reason: 'field "count(*)" not exist',
    code: 0
  },
  data: []
}
```

**Comment by yhmo** [comment, marked as answer]
`count(*) `is supported from v2.3. Your zilliz instance may be v2.2.x, it doesn't support `count(*)`.
